### ISSUE TITLE
Define map colours and settings in js (and maybe css)

### ISSUE THREAD

**Issue by peterdesmet** [inbo/vespa-watch#177]
Took me quite a while to find where the map colours are setting:

- Not in the css
- Not in the JS
- Not in the views
- But in the `base.py` settings:

https://github.com/inbo/vespa-watch/blob/6fd00074ac57d22e93b03862f934cf3434bc81ce/djangoproject/settings/base.py#L186-L208

I think it would make more sense to just set this in the JS (and if possible CSS for colours). @niconoe @bartaelterman thoughts?

**Comment by niconoe** [comment]
**TL;DR: the general mechanism behind this may be weird, but it's not that easy to find a better replacement. For the purely "visual/CSS" part, we can probably improve things by using CSS classes (detailed solution in the last paragraph).**

- The current mechanism works like this: a `js_config_object` custom template tag dumps a big JSON object in HTML templates, this object takes its data sources in various places (Django settings, URL configuration, ...) and those "JS settings" are consumed by the Vue components.
- It may be a bit weird (style not in CSS for example), but it is probably because it is a compromise between many (sometimes diverging) needs that are really transversal such as:
    - separate JS code and its configuration
    - avoid setting explicit JS configuration in each template (from exp. good at first, then quickly becomes messy) 
    - make URL endpoints work in all environments out of the box
    - give Python control on the JS configuration (needed for the previous point, among others)
    - keep "environment related" settings in Django settings files (to keep deployment simple)
    - I also wanted to do that as lightly/explicitly as possible, without adding more tools and dependencies

- So in general, I think before replacing this mechanism we should make sure the remplacement still check all the boxes above, or we risk throwing the 👶out with the 🛁.

- That being said, I think the current situation is indeed a bit "weird" and relatively easy to improve for the **visual part**  by:
    - keeping the general mechanism like it is
    - in the settings file, replace hard-coded values such as colors by CSS classes (`settings.MAP_CIRCLE_INDIVIDUAL_COLOR` -> `settings.MAP_CIRCLE_INDIVIDUAL_CLASS`)
    - define those CSS classes in the stylesheets
    - we'll also need update the JS code so it applies a class instead of a color. We might be restricted by the limitations of SVG CSS styling (to be investigated).

**Comment by peterdesmet** [comment]
I think we can close this issue.
